core/controller.go

```diff
@@ -268,6 +303,51 @@ func (c *Controller) StopStreamHandler(w http.ResponseWriter, r *http.Request, p
 	w.WriteHeader(http.StatusOK)
 }
 
+func (c *Controller) StartPreloadStream(Name string, Uri string) {
```

Comment:
I'm not convinced that this function should be exported. Also usually in Go names like URI or ID are capitalised. (I haven't seen this in other languages but it's a popular convention in the Go community)

Reply:
That is my lack of GoLang experience showing in multiple ways. My apologies. I must know 19 languages at this point, but I am completely green on GoLang at the moment.

Reply:
Can you help me understand how I made that exported? It appears the same as marshalValidatedURI to me, without the registered handler.

Reply:
If a function starts with a capital letter, it's exported 😄 That's the Golang way

Reply:
Oh, interesting. I completely missed that construct.